qemu/tests/ansible_test.py

```diff
@@ -0,0 +1,83 @@
+import os
+import json
+import shutil
+import logging
+
+from avocado.utils import process
+from avocado.utils import software_manager
+from virttest import error_context
+
+
+@error_context.context_aware
+def run(test, params, env):
+    """
+    Ansible playbook basic test:
+    1) Check ansible package exists
+    2) Launch the guest
+    3) Clone an ansible playbook repo
+    4) Generate the ansible-playbook command
+    5) Execute the playbook and verify the return status
+
+    :param test: QEMU test object
+    :param params: Dictionary with the test parameters
+    :param env: Dictionary with test environment.
+    """
+    sm = software_manager.SoftwareManager()
+    if not (sm.check_installed("ansible") or sm.install("ansible")):
+        test.cancel("ansible package install failed")
+
+    get_ip_timeout = params.get_numeric("get_ip_timeout")
+    guest_user = params["username"]
+    guest_passwd = params["password"]
+    ansible_callback_plugin = params["ansible_callback_plugin"]
```

Comment:
Would you please elaborate on why it is not optional?

Reply:
It can be an optional parameter but I set a default value (`ansible_callback_plugin = debug`) in the cfg.
The `debug` callback can let escape characters more readable from stderr/stdout, so I would like to highlight it in cfg, users can pick their like callback plugin.
```bash
$ ansible-doc -t callback -l | grep debug
debug                formatted stdout/stderr display           
```

Example output:
```
TASK [test-suite : Execute kar test suite] *************************************
fatal: [10.19.129.220]: FAILED! => {
    "changed": true,
    "cmd": "python3 ConfigTest.py --category=virtual_nic_device --guestname=RHEL.8.2.1 --customsparams=''",
    "delta": "21:59:53.681003",
    "end": "2020-06-04 15:51:43.209895",
    "rc": 1,
    "start": "2020-06-03 17:51:49.528892"
}

STDOUT:

Running bootstrap for qemu

1 - Checking the mandatory programs and headers
/usr/bin/xz OK
/usr/sbin/tcpdump OK
/usr/bin/nc OK
/usr/sbin/ip OK
/usr/sbin/arping OK
/usr/bin/gcc OK
/usr/include/asm/socket.h OK
/usr/include/asm/types.h OK
/usr/include/asm/unistd.h OK
/usr/include/python3.6m/Python.h OK
```

Reply:
ok, thanks, then I'd like to let it be an optional one with `ansible_callback_plugin = debug` as default, and can be removed if someone doesn't need this.